book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -1,31 +1,362 @@
+
+Where possible, try to arrange a recording of a talk or an alternative session plan that can be used in the event a speaker or facilitator is unable to attend. 
+You can also arrange to have speakers join remotely if it would be more supportive of their engagement. 
+
+`````{admonition} Case Study
+:class: tip
+The AI for Multiple Long Term Conditions Conference in April 2023 had a keynote speaker join and give their keynote address remotely in order to best support their involvement. 
+People with the lived experience of multiple long-term conditions may find travelling to events taxing which can have a negative impact on their energy levels and ability to engage in daily activities. 
+By supporting the speaker to speak remotely, they were able to give an engaging and thoughtful keynote address without the burden of having to travel. 
+`````
+
+(cl-inclusive-events-planning-GDPR)=
+### GDPR and Data Protection
+
+Data concerning health conditions and disability is considered "special category data" under GDPR. This also extends to cover a relative or carer of an individual with a health condition or disability. 
+Depending on the specific attendees and focus of your event, you may need to complete additional data protection forms and plan for how to manage the protected information. 
+This includes dietary information and disability status as disclosed in registration forms. Data management and privacy procedures will differ between organizations, 
+but it is important during planning to be aware that the information being collected may be special category and to establish a protocol for collecting and storing this information. 
+
+Please refer to your organization's policy on data protection and privacy and seek advice from your Data Protection Officer. 
+
+(cl-inclusive-events-planning-budget)=
+### Budget
+
+Attending and organizing events can place a large financial strain on all parties involved, and can cause the unintentional exclusion of many people with lived experience who may be engaging in projects as a volunteer. 
+This section highlights considerations that event organizers can make to increase the inclusion of an event, however, we would like to acknowledge that not all of these recommendations are possible to implement for every event. 
+Please work with your team and the resources available to address the most urgent needs of your attendees and use this section as a starting point for discussion.
+
+Some areas of consideration may be:
+
+(cl-inclusive-events-planning-tickets)=
+#### Ticket Pricing
+
+If your event is ticketed, ticket prices should be set thoughtfully and may consider having different tiers of pricing in addition to setting aside low-cost or complimentary tickets for carers and those experiencing financial hardship.
+
+(cl-inclusive-events-planning-honorarium)=
+#### Honorarium and Bursaries 
```

Comment:
Maybe also worth pointing out here that if the event is done within a large organisation it will require some policy work before the event 😂

Reply:
Yeah, I think just having that one sentence here makes sense to at least signpost the potential issue for now, and in the future there might be a different chapter on the topic of remuneration!